### ISSUE TITLE
View to see PR

### ISSUE THREAD

**Issue by mathieudutour** [kactus-io/kactus#11]
because you can't see the diffs as images on GitHub, it would be nice to be able to see PR in kactus

**Comment by mathieudutour** [comment]
maybe something like this:
![pr view](https://user-images.githubusercontent.com/3254314/28208655-b9032aaa-6886-11e7-942a-73f2928ec7bb.png)

**Comment by mathieudutour** [comment]
later on, we could even select an area on the visual diff and add a comment (using the github API and a special format to specify the area). The comments could then be shown on the diff directly

**Comment by pixelbandito** [comment]
Would you consider (also) adding an option to screenshot artboards and/or components and commit them in a directory alongside the pages? Then you'd get visual diffs in Github. Might also be more straightforward than building a PR view into the tool.

**Comment by mathieudutour** [comment]
that's what I was doing with [git-sketch-plugin](https://github.com/mathieudutour/git-sketch-plugin) but then you can't merge PRs, there are conflicts every time. So no, I don't think it's a nice solution, I prefer taking the time to do that correctly :)

**Comment by asos-dylanslewis** [comment]
Wouldn't this be better served by creating a [Danger](https://github.com/danger/danger) plugin that runs on CIs and posts a comment on the GitHub PR with before and after screenshots in a nice table? Would help with visibility to those who _only_ have GitHub available.

**Comment by mathieudutour** [comment]
I think it's a bit orthogonal. It would sure be nice although it requires a macOS CI server (Travis provides one for free). It would also require storing the screenshots somewhere. This is all possible but non-trivial to solve in a general case.

And I don't think it would be as nice as having it in Kactus where you could see the changes at different level, comment on a specific file, etc.

**Comment by asos-dylanslewis** [comment]
> This is all possible but non-trivial to solve in a general case.

Yeah, good point. I really wish the GitHub API allowed you to mimic dragging a file on a PR. So the best, private solution would be something like `fastlane`'s cert tool, where it stores the images in a separate, private repository on GitHub. But as you say, this is non-trivial.

**Comment by chriswiesner** [comment]
any plans on that?

i have a related question.
Merging a branch within kactus is not supported, right? So i'd need to merge using another git client and then go back to kactus to e.g. resolve merge conflicts, right?

**Comment by mathieudutour** [comment]
> Merging a branch within kactus is not supported

It is, `Branch > Merge into current branch`

**Comment by chriswiesner** [comment]
Thanks, sorry I somehow didn't pay attention to the menu, and was looking in the branch selection "tile"
